### DISCUSSION TITLE
French Translation Questions

### DISCUSSION THREAD

**Discussion by Onmyoujin** [Onmyoujin/combined-girl-mod#7] [General]
@Azurys I had a few questions about some inclusions later in your translation file, specifically additional strings from the strings.cs files.  Most of your translations for these strings are still gendered; is there an appropriate way to make phrases like "C'est ce qu'il préfère." gender-neutral in French?  Most of your inclusions appear to be the gender-specific variants of these types of generic phrases.

The reason I ask is these phrases get used by villagers to refer to random other villagers, so we can't actually be sure what the gender of the subject is if the mod is switching the genders of some villagers from male to female.  The solution I picked in English was to try to use gender-neutral language to avoid situations where villagers would use the wrong gendered pronouns to refer to their subject.

**Comment by Azurys** [comment]
Hi @Onmyoujin , first of all I'm sorry for this issue, I tried to do my best but didn't notice this detail. 
I actually tried to get every sentance gender-neutral because this is a real mess to manage when working on a non official content, but I struggled a bit. 
I will get this work redone when I can and give you a feedback as soon as possible. Sometimes it can be really tricky to get a correct gender neutral sentence which is not too long in french.

**Reply by Onmyoujin** [reply]
No worries!  If the responses have to be longer, that's fine; there's of course a maximum string length that will fit into the game's text boxes but it's pretty lengthy so feel free to be more verbose if it helps convey the meaning more properly.

I've included your translations in the latest release regardless since they will work as-is, but I'm looking forward to further improvements!
